src/main/java/com/savvato/tribeapp/services/ConnectServiceImpl.java

```diff
@@ -128,4 +125,16 @@ public ConnectOutgoingMessageDTO handleConnectionIntent(String connectionIntent,
         }
         return null;
     }
+
+    public boolean removeConnection(ConnectionRemovalRequest connectionRemovalRequest) {
+        if (Objects.equals(connectionRemovalRequest.requestingUserId, connectionRemovalRequest.connectedWithUserId)) {
```

Comment:
Is Object.equals() necessary here? Would == work as well?

Reply:
Since the ConnectionRemovalRequest DTO uses the Long wrapper type for the user IDs, they're technically different objects, so I don't think == would work. But if I use the "long" primitive type, then == should also work fine. (Other DTO objects also use Long wrapper types though, so for consistency, using the wrapper type may be better.)